buildconfig/BuildConfig.cpp.in

```diff
@@ -1,3 +1,36 @@
+/*
```

Comment:
How do you hold copyright on this file it's a set of macros

Reply:
Just look into Git blame. I have written the code below, that adds the current branch name to the version string.